recipes-bsp/tegra-binaries/tegra-binaries-32.1.0.inc

```diff
@@ -2,13 +2,21 @@ LICENSE = "Proprietary"
 LIC_FILES_CHKSUM = "file://nv_tegra/LICENSE;md5=2cc00be68c1227a7c42ff3620ef75d05 \
                     file://nv_tegra/LICENSE.brcm_patchram_plus;md5=38fb07f0dacf4830bc57f40a0fb7532e"
 
-SRC_URI = "${L4T_URI_BASE}/${L4T_BSP_PREFIX}_Linux_R${PV}_aarch64.tbz2"
-SRC_MD5SUM = "881fedee06a7446953a064b574227980"
-SRC_SHA256SUM = "e7ecaa32928484398d2492e6e9911262b4663e720b4867404d8fe1928fe483aa"
+SRC_URI = "${L4T_URI_BASE}/${L4T_BSP_PREFIX}_Linux_R${PV}_aarch64.tbz2;name=l4t \
+           ${NVIDIA_DEVNET_MIRROR}/secureboot_r32.1.tbz2;name=sb"
+L4T_MD5SUM = "881fedee06a7446953a064b574227980"
+L4T_SHA256SUM = "e7ecaa32928484398d2492e6e9911262b4663e720b4867404d8fe1928fe483aa"
+SB_MD5SUM = "8c61778838533cc20a9a248882091a46"
+SB_SHA256SUM = "f1b8dedf6544c0588997430333553a5b3973a0b826cf21771d84eabce50a31d6"
+SRC_URI[l4t.md5sum] = "${L4T_MD5SUM}"
+SRC_URI[l4t.sha256sum] = "${L4T_SHA256SUM}"
+SRC_URI[sb.md5sum] = "${SB_MD5SUM}"
+SRC_URI[sb.sha256sum] = "${SB_SHA256SUM}"
+
+# L4T 32.1 does not support Jetson Nano secureboot (L4T 32.2 support Jetson Nano secureboot)
+SRC_URI_tegra210 = "${L4T_URI_BASE}/${L4T_BSP_PREFIX}_Linux_R${PV}_aarch64.tbz2;"
```

Comment:
I don't think this is going to work - SRC_URI checksums won't be there as needed.  We might need some python to handle this.

Reply:
Sorry for the delay. I'm sorry I did not get what you mean?

Reply:
This patch is replacing the single-element SRC_URI with a multiple-element one, but with the `tegra210` overrides you aren't matching up with the new structure.

You need to add `name=l4t` to this line to match up with the `SRC_URI[l4t.md5sum]` and `SRC_URI[l4t.sha256sum]` settings, and you need to rename the settings in the next two lines to be `L4T_MD5SUM_tegra210` and `L4T_SHA256SUM_tegra210` as well.

Reply:
Got it I will change accordingly. Thank you!

Reply:
For tegra210 I think SRC_URI is replaced by the following, because `SRC_SHA256SUM_tegra210` I thought bitbake should ignore `L4T_MD5SUM`. Is my assumption correct:

```
SRC_URI_tegra210 = "${L4T_URI_BASE}/${L4T_BSP_PREFIX}_Linux_R${PV}_aarch64.tbz2"
SRC_MD5SUM_tegra210 = "15fda16d83e9a189ea5534108f57af2f"
SRC_SHA256SUM_tegra210 = "e4902a952245217227486322b3e8cef47942f87f2ea91f9e67d700314ae3ade3"
```

Reply:
No, because we aren't setting `SRC_URI[md5sum]` and `SRC_URI[sha256sum]`.  I don't think you can put overrides on flag settings like that.  That's why I recommended the other approach.

Reply:
Thank you I will change it